ci-operator/config/redhat-operator-ecosystem/cvp/redhat-operator-ecosystem-cvp-ocp-4.7.yaml

```diff
@@ -11,6 +11,44 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: cvp-common-generic-claim
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: cvp
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.7"
+  steps:
+    dependency_overrides:
+      BUNDLE_IMAGE: brew.registry.redhat.io/rh-osbs-stage/e2e-e2e-test-operator-bundle-container:8.0-3
+      INDEX_IMAGE: brew.registry.redhat.io/rh-osbs-stage/iib:23576
+    env:
+      BASE_DOMAIN: cpaas-ci.devcluster.openshift.com
+      CHANNEL: "4.3"
+      INSTALL_NAMESPACE: '!create'
+      PACKAGE: e2e-test-operator
+      PYXIS_URL: ""
+      TARGET_NAMESPACES: '!install'
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
```

Comment:
No, this does not make sense. `ipi-aws-pre` and `ipi-aws-post` are chains that contain steps that install and deprovision clusters, they will not work with claimed clusters. What you need to do is to take what the `generic-claim` workflow has in `pre` and `post`, and simply add CVP-specific steps to it (similar to how the existing CVP workflows just add more steps to what e.g. `ipi-aws` has).

Basically just take all the CVP steps here and move them to the workflow. Then the test here will not need to specify any pre/test/post, exactly how the other tests (that use the existing workflows) do not set them.

Reply:
Thats exactly what I did previously and it started to fail on the same exact thing as I was mentioning earlier yesterday.

Reply:
Yes, that was exactly the approach I did yesterday and it was failing on unbound variables.

Reply:
But lets see how this one goes.

Reply:
I see, as you said 
ipi-aws-pre and ipi-aws-post are chains that contain steps that install and deprovision clusters, they will not work with claimed clusters!!!!
This was probably causing the issue to occur. Lets try it without those chains.